flutter/README.md

```diff
@@ -8,6 +8,16 @@ tagline: |
 To update or switch versions, run `webi flutter@stable` (or `@v2.2`, `@beta`,
 etc).
 
+### Files
+
+These are the files / directories that are created and/or modified with this install:
+
+```text
+~/.config/envman/PATH.env
+~/.local/opt/flutter
+~/.local/opt/flutter-{version}
```

Comment:
Flutter probably has a config file either at the home directory or project level.

Please figure out what that is, and place it here as well.

For example, that might look like this:

```text
~/flutter-conf-thing.json
<PROJECT-DIR>/flutter-conf-thing.json
```

- `<PROJECT-DIR>` would be literal (instead of `~/`).
- `flutter-conf-thing.json` would be whatever the config file or package file is

Reply:
It's project base, I've changed it accordingly.